packages/components/src/Heading/Heading.tsx

```diff
@@ -14,23 +15,26 @@ export interface LevelMap {
 }
 
 export function Heading({ level = 5, children }: HeadingProps) {
+  const config = getAtlantisConfig();
+  const inRetheme = config?.JOBBER_RETHEME;
+
   const levelMap: LevelMap = {
     1: {
       element: "h1",
       size: "jumbo",
-      fontWeight: "black",
+      fontWeight: inRetheme ? "extraBold" : "black",
```

Comment:
@darryltec H1 needs to swap over to Inter as well

Reply:
It is inter haha 
![image](https://github.com/GetJobber/atlantis/assets/15986172/5ab33141-faa6-43e6-aae5-aca3073c1ab7)

Reply:
Maybe I'm doing something wrong, H2/3 changed but H1 is still showing as Dispaly
![image](https://github.com/GetJobber/atlantis/assets/10064579/d152d964-13e8-4cda-87ce-2889255228b4)

Reply:
oh yah, that's the part where you'd need `jobber-retheme` on the body

```
setTimeout(() => {
  document.body.classList.add("jobber-retheme");
}, 0);

// @ts-expect-error
window.ATLANTIS_CONFIG = {
  JOBBER_RETHEME: true,
}
```

Reply:
Should just the H1 be affected by that? The others change without the class applied.

Reply:
yeah, it's due to this weird thing where boldness is tied to font 
![image](https://github.com/GetJobber/atlantis/assets/15986172/d1958eb9-65cc-41e6-97f1-e326ae6d8833)